base/interactiveutil.jl

```diff
@@ -566,7 +566,7 @@ end
 Return an array of methods with an argument of type `typ`.
 
 The optional second argument restricts the search to a particular module or function
-(the default is all modules, starting from Main).
+(the default is all modules).
```

Comment:
New implementation doesn't do this anymore. I'm not against making it recursive, but it should be consistent, and possibly in a separate commit / PR.